### PR TITLE
Create a root_VS2015 corresponding to the Visual Studio 2015 (2)

### PR DESCRIPTION
**Pull request by daisukenishino**
- Create a root_VS2015 corresponding to the Visual Studio 2015.  
https://github.com/OpenTouryoProject/OpenTouryo/issues/113
  - Pull Request https://github.com/OpenTouryoProject/OpenTouryoTemplates/pull/18 has been fixed.

### CODE REVIEW

File: root_VS2015/files/resource/Log/Examples of rolling of date+size.xml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+		<filter type="log4net.Filter.LevelRangeFilter">
+			<levelMin value="DEBUG" />
+			<levelMax value="FATAL" />
+		</filter>
+	</appender>
+
+－－－
+
+Log4Netのちょっと良い設定
+http://www.ailight.jp/blog/koido/archive/2005/03/09/4871.aspx
```
Comment (yusukemaegawa) [inline root_VS2015/files/resource/Log/Examples of rolling of date+size.xml]: URLが違います

File: root_VS2015/files/resource/Log/Examples of rolling of date+size.xml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+いつも参考にさせてもらっています。
```
Comment (yusukemaegawa) [inline root_VS2015/files/resource/Log/Examples of rolling of date+size.xml]: ブログのコメントをそのまま張るのであれば、（引用）と一言つけてください。
また、この文章がある理由はなんでしょうか。（単なる備忘録？）

**Comment by yusukemaegawa** [comment]
The debug of "WSClientSL_sample" and "WSClientWinStore_sample" fails.
- Because the url of web service contains the name of virtual directory, the access to web service fails.
    - http://localhost:9996/**WSClientSL_sample**/WebService/ServiceForMu.asmx

File: root_VS2015/files/resource/Log/Examples of rolling of date+size.xml (hunk 2 above)
Comment (daisukenishino) [inline root_VS2015/files/resource/Log/Examples of rolling of date+size.xml]: @yusukemaegawa 
上記2点、01-71版リリース時に修正しておきます。

**Comment by daisukenishino** [comment]
@yusukemaegawa 
> https://github.com/OpenTouryoProject/OpenTouryoTemplates/pull/20#issuecomment-154257668

I have add the above pointed out to the following issue as a correction target.

- [The debug of "ProjectX_sample" in "root_VS2013" fails.  - Issue 12](https://github.com/OpenTouryoProject/OpenTouryoTemplates/issues/12#issuecomment-154292428)

**Pull request merged**